README.md

```diff
@@ -90,6 +90,7 @@ More libraries are coming soon (the overall list of items we currently plan to m
 |**System.Linq.Parallel**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Linq.Parallel.svg)](https://www.myget.org/gallery/dotnet-core)|<sub> Provides a parallelized implementation of LINQ to Objects. "Parallel LINQ" (PLINQ) implements the full set of LINQ standard query operators as well as additional operators specific to parallel operations.</sub>|
 |**System.Linq.Queryable**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Linq.Queryable.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides LINQ standard query operators that operate on objects that implement ```IQueryable<T>```.</sub>|
 |**System.Net.Http**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Net.Http.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides a programming interface for modern HTTP applications, including HTTP client components that allow applications to consume web services over HTTP and HTTP components that can be used by both clients and servers for parsing HTTP headers.</sub>|
+|**System.Net.Http.WinHttpHandler**<br/>[![MyGet Package](https://img.shields.io/myget/dotnet-core/v/System.Net.Http.WinHttpHandler.svg)](https://www.myget.org/gallery/dotnet-core)|<sub>Provides a message handler for HttpClient based on the WinHTTP interface of Windows.  While similar to HttpClientHandler, it provides developers more granular control over the application's HTTP communication than the HttpClientHandler. This allows developers to implement more advanced HTTP scenarios or modify system defaults (for example, proxy settings, timeouts and server SSL certificate validation).</sub>|
```

Comment:
Could we trim this down to just be the first sentence: "Provides a message handler for HttpClient based on the WinHTTP interface of Windows."?  The latter sentences feel like useful info but more than is needed for a brief description of a component.  It's also not accurate for non-Windows systems, since the component is explicitly tied to Windows.